shared/common-adapters/standard-screen.js.flow

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+import {Component} from 'react'
+
+export type Props = {
+children?: ?(React$Element<*> | Array<React$Element<*>>),
```

Comment:
Flow's default react types have `children: ?any`. I'm going to match that typing.